domain/schema/model/sql/0015-charm.sql

```diff
@@ -116,6 +116,15 @@ SELECT
 FROM charm_origin AS co
 LEFT JOIN charm_source AS cs ON co.source_id = cs.id;
 
+CREATE VIEW v_charm_annotation AS
+SELECT
+    c.uuid,
+    cm.name,
+    co.revision
+FROM charm AS c
```

Comment:
Shouldn't we also have the `charm_source.name` to make it unique?

Reply:
We can add that when we punt through all the changes.